compiler/bcq-tools/generate_bcq_output_arrays

```diff
@@ -87,7 +87,154 @@ def find_bcq_version(flags):
 
 
 def print_bcqinfo_output_arrays_v1(flags):
-    pass
+    """
+    This function generates a file which includes output arrays of BCQ v1
+    information bundles. Each bundle is consisted with one of candidate
+    operations (BCQ may be applied) and BCQ constant nodes related with
+    the operation.
+    """
+    graph = load_graph(flags.input_path)
+    graph_def = graph.as_graph_def()
+    ops = graph.get_operations()
+
+    # If there is a constant node named PREFIX_1/bcqinfo_alpha,
+    # it is used for applying BCQ to constant node named PREFIX_1.
+    # Collected prefixes will be used for connecting
+    # bcqinfo nodes and prefix node user operations.
+    prefix_set = set()
+    has_dequant_weight = False
+    for op in ops:
+        if op.type == "Const" and "/bcqinfo_" in op.outputs[0].name:
+            # Metadata do not have prefix
+            if "one_compiler/bcqinfo_one_metadata" in op.outputs[0].name:
+                continue
+
+            prefix_index = op.outputs[0].name.index("/bcqinfo_")
+            prefix = op.outputs[0].name[:prefix_index]
+            prefix_set.add(prefix)
+
+            infoname = op.outputs[0].name[prefix_index + 1:-2]
```

Comment:
there are several `-2` here and below. can you please add a comment about `-2` ?

Reply:
FYI, `op.outputs[0].name` has format like `nodename:0`. `-2` removes `:0`.